tide_core.install

```diff
@@ -1446,3 +1446,60 @@ function tide_core_update_8061() {
   $field_storage->setSetting('allowed_values', $allowed_values);
   $field_storage->save();
 }
+
+/**
+ * Update table of contents and display headings fields.
+ */
+function tide_core_update_8063() {
+  // Remove redundant existing listing key values from DB.
+  $display_headings_tables = [
+    'node__field_node_display_headings',
+    'node_revision__field_node_display_headings',
+  ];
+  foreach ($display_headings_tables as $table) {
+    $query = \Drupal::database();
+    if ($query->schema()->tableExists($table)) {
+      $query = $query->delete($table);
+      $orGroup = $query->orConditionGroup()
+        ->condition('field_node_display_headings_value', 'showH2AndAccordionTitle')
+        ->condition('field_node_display_headings_value', 'showH2AndH3AndAccordionTitle');
+      $query->condition($orGroup);
+      $query->execute();
```

Comment:
As these fields already has value in it and they want to remove these two options that was added in the last release. The best way to avoid field migration and also not change the existing values for the first two radio buttons is just to clean up the redundant values from db if table exists. Then do the storage config updates for the field.